runtime/list.go

```diff
@@ -291,6 +300,29 @@ func listNE(f *Frame, v, w *Object) (*Object, *BaseException) {
 	return listCompare(f, toListUnsafe(v), w, NE)
 }
 
+func listPop(f *Frame, args Args, kwargs KWArgs) (*Object, *BaseException) {
+	argc := len(args)
+	expectedTypes := []*Type{ListType, IntType}
+	if argc == 1 {
+		expectedTypes = expectedTypes[:1]
+	}
+	if raised := checkMethodArgs(f, "pop", args, expectedTypes...); raised != nil {
+		return nil, raised
+	}
+	i := -1
+	if argc == 2 {
+		i = toIntUnsafe(args[1]).Value()
```

Comment:
Oh! Good catch. I just assumed. If that's the way it works in CPython then it's fine to leave it out. However, you still need to support `long`:

```
>>> [1,2,3].pop(2L)
3
```